README.md

```diff
@@ -15,6 +15,7 @@ This exporter currently supports these monitor types:
 - URL ["Website"](https://www.site24x7.com/help/admin/adding-a-monitor/website-monitoring.html)
 - HOMEPAGE ["Web Page Speed (Browser)"](https://www.site24x7.com/help/admin/adding-a-monitor/web-page-analyzer.html)
 - REALBROWSER ["Web Transaction (Browser)"](https://www.site24x7.com/help/admin/adding-a-monitor/webapplication-monitoring-realbrowser.html)
+- RESTAPI ["REST API"](https://www.site24x7.com/help/admin/adding-a-monitor/rest-api-monitor.html)
```

Comment:
Put this in the same location as the enum variant so it's easy to spot the mapping at a glance.